### ISSUE TITLE
This plugin messes up metadata generated by Grav Core

### ISSUE THREAD

**Issue by bricebou** [clivebeckett/grav-plugin-admin-addon-media-metadata#11]
Hi,

This is a great plugin except it doesn't work along core Grav media management...

If I don't activate the plugin, the metadata file is generated by Grav :

```
aperture: f/2.2
camera: 'iPhone 6s'
Orientation: 1
horizontalResolution: 72
verticalResolution: 72
software: 13.4.1
exposure: 1/17
iso: 250
creationdate: 2020-04-26T16:44:29+02:00
focalLength: 4.15
ColorSpace: 1
gps: '46.086472222222,1.4754138888889'
```

But if your plugin is activated, these metadata are not written in this file, only the title, alt and caption fields...

**Comment by clivebeckett** [comment]
Thanks for the note. I reckon this is when the setting `media.auto_metadata_exif` is set to true – I had only very recently learned about this feature anyway.

I guess this should be solveable. After all, it should work for files that already had their meta.yaml file created before the plugin was installed as it will only add to the data and not delete any. This should be possible for newly uploaded images as well.

**Comment by bricebou** [comment]
That's great news :-)
Thanks for your fast answer !

**Comment by clivebeckett** [comment]
Since [v1.1.0](https://github.com/clivebeckett/grav-plugin-admin-addon-media-metadata/tree/v1.1.0) this problem has been solved … with a small issue in Admin 1.10 however: you need to save the page before adding any other metadata using my plugin – but if you take care of this, the plugin will work. In Admin 1.9 this problem does not exist.

### Explanations

**Admin 1.9:**
The meta.yaml with the EXIF data will be written immediately upon upload. My plugin does now not write empty meta.yaml files anymore so it does not get in the way of this system functionality. If you write metadata using the form provided by my plugin, it will add this metadata to the meta.yaml and keep all the data that is already in there.

**Admin 1.10:**
The meta.yaml with the EXIF data will only be written upon saving the page. If you write metadata using the form provided by my plugin before the page is saved, a meta.yaml will be created by my plugin and the EXIF data will not be written anymore. That’s why you need to save the page before writing metadata using my plugin.

I will close this issue and open a new one for solving the problem in Admin 1.10.
